dev/com.ibm.ws.threading/test/com/ibm/ws/threading/internal/PolicyExecutorTest.java

```diff
@@ -76,24 +76,34 @@ public Integer call() throws Exception {
     public void testExpedite() throws Exception {
         ExecutorServiceImpl globalExecutor = new ExecutorServiceImpl();
         Map<String, Object> globalExecutorConfig = new HashMap<String, Object>();
-        globalExecutorConfig.put("coreThreads", 1);
-        globalExecutorConfig.put("maxThreads", 1);
-        globalExecutorConfig.put("keepAlive", TimeUnit.MINUTES.toMillis(5));
+        globalExecutorConfig.put("coreThreads", ExecutorServiceImpl.MINIMUM_POOL_SIZE);
+        globalExecutorConfig.put("maxThreads", ExecutorServiceImpl.MINIMUM_POOL_SIZE);
+        // Jan-2019: remove keepAlive, no longer supported by the default executor
+        // globalExecutorConfig.put("keepAlive", TimeUnit.MINUTES.toMillis(5));
```

Comment:
Remove commented-out code and historical comment from test case.  The only reason for having the code in this test would have been to simulate what the actual Liberty global executor instance does.

Reply:
Agreed, done.